src/Adapter/Attribute/Repository/AttributeRepository.php

```diff
@@ -82,4 +84,128 @@ public function assertAllAttributesExist(array $attributeIds): void
             throw new AttributeNotFoundException('Some of provided attributes does not exist');
         }
     }
+
+    /**
+     * @param ProductId $productId
+     * @param LanguageId $langId
+     *
+     * @return array[]
+     */
+    public function getAttributesInfoByProductId(ProductId $productId, LanguageId $langId): array
```

Comment:
Should me move this service into the Product domain? (it's new in 178)
Like in `namespace PrestaShop\PrestaShop\Adapter\Product\AttributeGroup\Attribute\Repository`